gravitee-apim-console-webui/src/components/documentation/edit-tabs/edit-page-configuration.html

```diff
@@ -81,6 +81,14 @@ <h3>Swagger</h3>
     <md-checkbox ng-disabled="$ctrl.readOnly" ng-model="$ctrl.page.configuration.tryIt" ng-true-value="'true'" ng-false-value="'false'">
       Enable "Try it!" mode. Perhaps you will have to configure the CORS in the proxy section.
     </md-checkbox>
+    <md-checkbox
+      ng-disabled="$ctrl.readOnly"
+      ng-model="$ctrl.page.configuration.disableSyntaxHighlight"
+      ng-true-value="'true'"
+      ng-false-value="'false'"
+    >
+      Disable Styling for large payload json data
```

Comment:
Maybe we can specify that it's only for the response body here and not the entire swagger styling